src/app/components/Button.js

```diff
@@ -5,6 +5,7 @@ export const Button = (
     classList: [],
     onClickFn: undefined,
     icon: '',
+    isSpecial: '',
```

Comment:
Tutaj to bardziej false jest defaultowo :) Niż ''

Reply:
racja! poprawię!